custom/icds_reports/management/commands/fix_missing_thr_forms.py

```diff
@@ -0,0 +1,118 @@
+from datetime import datetime
+
+from dateutil.relativedelta import relativedelta
+from django.core.management import BaseCommand
+from django.db import connections
+
+from corehq.apps.locations.models import SQLLocation
+from custom.icds_reports.const import (
+    DASHBOARD_DOMAIN
+)
+from custom.icds_reports.tasks import (
+    icds_state_aggregation_task
+)
+
+query_text_1 = """
+    UPDATE "agg_awc_{month}_5" t
```

Comment:
I don't think this will work, you are trying to update a local table from a distributed table.